requirements/base.txt

```diff
@@ -309,7 +310,7 @@ vine==5.0.0
     #   kombu
 wcwidth==0.2.5
     # via prompt-toolkit
-werkzeug==2.1.2
+werkzeug==2.3.3
```

Comment:
I suggest to pin `werkzeug` in setup.py because we use this lib in superset project directly.

Reply:
done, makes sense